ports/directxsdk/vcpkg.json

```
@@ -1,7 +1,18 @@
 {
   "name": "directxsdk",
-  "version-string": "jun10",
-  "description": "DirectX SDK",
-  "homepage": "https://www.microsoft.com/en-us/download/details.aspx?id=6812",
-  "supports": "windows & !arm & !uwp"
+  "version-string": "jun10b",
```

```suggestion
  "version-string": "jun10",
  "port-version": 1,
```

This isn't really a new version of the SDK, just a change for the port to actually work.

---

Actually, I would recommend the version update. This version of the port doesn't just blindly install everything in the DXSDK like the previous version did. It installs a specific subset of files and provides opt-in features for others.

---

The new version is `jun10#1` instead of `jun10` - this is done by setting the `port-version` as shown above. The `version-string` should only change if the upstream provider is responsible for the update. See the [documentation](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/control-files.md#port-version) for more details.

---

OK...